react-router.config.ts

```diff
@@ -17,4 +26,7 @@ export default {
   async prerender() {
     return ['/terms', '/privacy', '/mint-risks'];
   },
+  future: {
+    unstable_middleware: true,
```

Comment:
needed for reason explained here https://github.com/MakePrisms/boardwalkcash/pull/584/files#r2291755340